### ISSUE TITLE
Alias replaces HTML brackets to HTML codes, e.g  "<img src=..." to  "&lt;img src="

### ISSUE THREAD

**Issue by AlexWorx** [doxygen/doxygen#10747]
Summary
======================================================
Doxygen replaces (sometimes?) characters "<" and ">" in macros to HTML "&lt;" and ""&gt;"

Or: I have to phrase my ALIAS now differently after years where it worked? 
I did not find any hint to that.   I tried some things, but I could not manage.

Steps how I created this sample:
======================================================
(but all files attached)
- run

        doxygen -g doxygen.ini

- Add an alias like this to the ini file:

        ALIASES  = "https{2}=[\1 <img src=\"../link.svg\" height=\"12\" width=\"10\">](https://\2)"

- Add source to the ini file:

   INPUT                  = main.cpp

- Added attached files "main.cpp" and "link.svg" to the folder

Steps to reproduce the bug:
======================================================
- Run doxygen:

        doxygen doxygen.ini

- Open file

         html/html/classtest.html

  in a browser and in a text editor

- You will see a broken text link. Doxygen replaces characters "<" and ">" from the "<img src=..>" tag
  with HTML syntax "&lt;" and ""&gt;"

  Precisely, This is generated:

      <a href="https://alib.dev">Test Link &lt;img src="../link.svg" height="12" width="10"&gt;</a>

  but this should be generated:

      <a href="https://alib.dev">Test Link <img src="../link.svg" height="12" width="10"></a>

Additional Notes:
======================================================
- Similar other macros work fine, for example:

          "spantest{1}=<span>\1</span>"

  does not replace the lt/gt to html codes

- I had used this "https" macro successfully since years.
  The new behavior was introduced with version 1.9.3 or 1.9.2. (I do not have 1.9.2 available)
  1.9.1 still works.

[doxygenBugLtGtInAliases.zip](https://github.com/doxygen/doxygen/files/14632624/doxygenBugLtGtInAliases.zip)

**Comment by albert-github** [comment]
I did a few tests
- the version it changed was indeed 1.9.3 and to be more specific the pull request #8961 (for the issue #8956). The issue #8965 though looks like at first glance to be geared to the `#` in the `(...)` part and here we have a problem in the `[...]` part. but there is also a reference to the `\ref` command that needs to have an pair of `"`  around its text, maybe the `"` adding should only be done in case of a translation to a `\ref` command.
  In the 1.9.2 version the markdown conversion gave (`doxygen -d markdown`):
  ```
  <a href="https://alib.dev" >Test Link <img src="../link.svg" height="12" width="10"></a>
  ```
  in the 1.9.3 version the result was
  ```
  <a href="https://alib.dev" >Test Link <img src=\"../link.svg\" height=\"12\" width=\"10\"></a>
  ```
  This has to be checked further.
- You wrote: 
  > I have to phrase my ALIAS now differently after years where it worked?
I did not find any hint to that. I tried some things, but I could not manage.

  I don't think there is anywhere a hint on how to do it , though I found a way around:
  ```
  ALIASES                = "https{2}=[\1 <img src='../link.svg' height='12' width='10'>](https://\2)"
  ```

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #10748

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.11.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
